packages/Tag/src/mixins.scss

```diff
@@ -0,0 +1,30 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+
+@mixin ids-tag-mixin-decoration(
+  $backgroundColor,
+  $color,
+  $borderColor: transparent
+) {
+  background: $backgroundColor;
+  border: 1px solid $borderColor;
+  color: $color;
+}
+
+@mixin ids-tag-mixin-size($fontSize, $verticalPadding, $horizontalPadding) {
+  font-size: $fontSize;
+  padding: $verticalPadding $horizontalPadding;
+
+  .ids-tag__icon {
+    margin:
```

Comment:
well it's because I needed a negative margin since when there are icons, there is less space on the sides. This seemed to be the easiest option. The right margin could have been done like that. but there is no space token variable that is 0.6rem.